src/app/api/quality_matrix/collections.py

```diff
@@ -66,8 +66,29 @@ async def id_to_title_mapping(node_id: uuid.UUID) -> dict[str, str]:
 
 async def collection_quality(
     node_id: uuid.UUID, match_keyword: str = "path"
-) -> QUALITY_MATRIX_RETURN_TYPE:
+) -> tuple[list[QualityOutput], {dict[str, int]}]:
```

Comment:
This type annotation makes no sense to me! I assume the`{}` need to go?

Reply:
Yes! I wonder why this even worked at all. Refactored